e2e/testdata/fn-eval/missing-fn-config/.expected/config.yaml

```diff
@@ -16,5 +16,3 @@ testType: eval
 exitCode: 1
 image: gcr.io/kpt-fn/set-namespace:v0.1.3
 stdErr: "failed to configure function: input namespace cannot be empty"
-stdOut: |
-  [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3"
```

Comment:
So this is not part of stderr?

Reply:
This will be part of stderr, but we do only the substring match here. We only check for the message relevant to the test.

There are other tests which check of this is included in stderr. We need not check that in all the tests.

Reply:
Link to tests that assert the non-structure output for:

- eval
- eval | eval
- source | eval
- render
- render | eval

Reply:
@droot @Shell32-Natsu

Reply:
@frankfarzan Added third commit which covers all these cases.